doc/devel/dependencies.rst

```diff
@@ -176,9 +178,8 @@ remember to clear your artifacts before re-building::
 
   git clean -xfd
 
-
-Manual Download
-^^^^^^^^^^^^^^^
+Install from source
```

Comment:
Not quite sure whether "install" is the right term here. But I don't have an alternative either.

Reply:
I don't want to over-think this, but "source" is a quite generic term. IMHO good headings would be
- C libraries
  - Automatic download (*this section header is missing but should be added*)
  - Use system libraries
  - From source files (or "Custom source files")